config/initializers/ams.rb

```diff
@@ -0,0 +1 @@
+ActiveModelSerializers.config.adapter = :json_api
```

Comment:
We use `:json` as a serializer. JSONApi is a standard for APIs, but we don't use it.